packages/storybook-addon/src/utils/path.ts

```diff
@@ -0,0 +1,21 @@
+import os from 'os';
+import path from 'path';
+
+export const correctImportPath = (context: string, entryFile: string) => {
```

Comment:
I think it's best if this function is moved to `utilities` package. I already see other use cases for this.

Reply:
This util is used to get the import path in the async boundary(index bootstrap). Do you think it's a good idea to share the util? I'm not seeing cases when it will be needed

Reply:
@pavandv I believe you're right, `correctImportPath` is something that can be more widely used across the MF ecosystem as its something that we have to deal with in more places.

Reply:
done ✅